source/openapi/pom.xml

```diff
@@ -128,7 +128,31 @@
                             <generateApiTests>false</generateApiTests>
                             <configOptions>
                                 <modelPackage>com.openshift.cloud.api.models</modelPackage>
-                                <apiPackage>com.openshift.cloud.api</apiPackage>
+                                <apiPackage>com.openshift.cloud.api.kas</apiPackage>
```

Comment:
Is model package ok here. it is the same for both

Reply:
Good catch